docs/src/api/class-browser.md

```diff
@@ -256,6 +256,9 @@ await browser.CloseAsync();
 ### option: Browser.newContext.proxy = %%-context-option-proxy-%%
 * since: v1.8
 
+### option: Browser.newContext.clientCertificates = %%-context-option-clientCertificates-%%
```

Comment:
launchPersistentContext sounds good but maybe omit it from Android for now? Since Android runs in an Emulator, so networking wise it might be hard to expose the proxy into it.

Reply:
Done.